contrib/python-acme/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "python-acme"
+pkgver = "2.10.0"
+pkgrel = 0
+build_wrksrc = "acme"
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-wheel",
+]
+depends = [
+    "python-cryptography",
+    "python-josepy",
+    "python-openssl",
+    "python-pyrfc3339",
+    "python-pytz",
+    "python-requests",
+    "python-setuptools",
```

Comment:
this belongs in hostmakedepends

Reply:
its in install requires in setup.py

https://github.com/certbot/certbot/blob/2a41402f2ade41b41532e11f29b295eefc3e1552/acme/setup.py#L8

Reply:
the code doesn't import setuptools or pkg_resources though

Reply:
okay will move it